### ISSUE TITLE
Compile Error In PaletteView at line 33 -> "if let current Palette = ..."

### ISSUE THREAD

**Issue by Noemata** [markusmoenig/Fabricated#1]
Getting this compile error:

xxx/Fabricated-main/Shared/PaletteView.swift:33:24 Initializer for conditional binding must have Optional type, not 'TileSetPalette'

**Comment by markusmoenig** [comment]
Fixed with the latest commit, thanks.

**Comment by Noemata** [comment]
No, thank you.  Great work!  I did have minor build issues with a couple other repos.  I'm guessing few have looked into things in detail.

**Comment by markusmoenig** [comment]
Problem is I never finished some apps, like Fabricated and Signed. So they are not on the AppStore. I am working on new apps which will replace them.

**Comment by Noemata** [comment]
Even as is, these are some of the best usage examples on GitHub.  I was lucky to stumble across your repo.  Just one more question, I'm thinking of switching to iOS 17 because of the metal related enhancements.  Specifically, being able to easily attach shader effects to a view.  I wasn't able to find anything that worked this seamlessly for previous iOS releases (mostly doing tvOS work).  Is there anything good out there, or should I just switch over since I won't be releasing a product until the fall?  I'm inclined to go with the latter.

**Comment by markusmoenig** [comment]
Yeah I would go with the later. I don't think you will be able to get this to work in earlier versions.
